### PR TITLE
updated all readme files

### PR DESCRIPTION
**Pull request by OfriRosenbaum**

### CODE REVIEW

File: src/main/java/frc/robot/drivetrains/advancedtankdrivetrain/README.md [hunk 1]
```diff
@@ -1 +1,8 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Drivetrain](Drivetrain.java)
+- Change the PID, FeedForward and distance per pulse in [Drivetrain](Drivetrain.java)
+- Change the type of gyro in [Drivetrain](Drivetrain.java)
+
 ![drivetrain](https://imgur.com/dfjdcg6.png)
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/drivetrains/advancedtankdrivetrain/README.md:8]: put the picture first

File: src/main/java/frc/robot/drivetrains/simpletankdrivetrain/README.md [hunk 2]
```diff
@@ -1 +1,6 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Robot](Robot.java)
+
 ![drivetrain](https://imgur.com/dfjdcg6.png)
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/drivetrains/simpletankdrivetrain/README.md:6]: put the picture first

File: src/main/java/frc/robot/subsystems/climber/README.md [hunk 3]
```diff
@@ -1 +1,6 @@
-![climber arm](https://imgur.com/J6VwdEP.png)
\ No newline at end of file
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Climber](Climber.java)
+
+![climber arm](https://imgur.com/J6VwdEP.png)
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/subsystems/climber/README.md:6]: put the picture first

File: src/main/java/frc/robot/subsystems/gripper/README.md [hunk 4]
```diff
@@ -1,2 +1,7 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Gripper](Gripper.java)
+
 ![empty gripper](https://imgur.com/vyKXnYx.png)
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/subsystems/gripper/README.md:6]: I think you get the point

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,11 +1,14 @@
 # SpikesLib - Examples
+
 Examples of how to use the [SpikesLib](https://github.com/Spikes-2212-Programming-Guild/SpikesLib2) library.
 
 ## Includes:
+
 - Several subsystems and drivetrains, each with its own implementation example.
 - Each subsystem has a RobotMap class, which holds the ports for the robot's components.
-  - Each interface in RobotMap represents a different type port.
+    - Each interface in RobotMap represents a different type port.
```
Comment (Yoel-Perman-Brilliant) [inline README.md]: type of port

File: src/main/java/frc/robot/drivetrains/advancedtankdrivetrain/README.md [hunk 6]
```diff
@@ -1 +1,8 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Drivetrain](Drivetrain.java)
+- Change the PID, FeedForward and distance per pulse in [Drivetrain](Drivetrain.java)
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/drivetrains/advancedtankdrivetrain/README.md]: PID constants, Feed Forward gains

File: src/main/java/frc/robot/subsystems/shooter/README.md [hunk 7]
```diff
@@ -1 +1,7 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Shooter](Shooter.java) (if you change this, you __must__ also change the
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/subsystems/shooter/README.md]: Not necessarily must, They might be using a falcon which also has a talon encoder

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/drivetrains/advancedtankdrivetrain/README.md (hunk 1 above)
Comment (OfriRosenbaum) [reply src/main/java/frc/robot/drivetrains/advancedtankdrivetrain/README.md:8]: no

File: src/main/java/frc/robot/subsystems/shooter/README.md [hunk 8]
```diff
@@ -1 +1,7 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Shooter](Shooter.java) (if you change this, you may also need change the
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/subsystems/shooter/README.md]: need to change

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/shooter/README.md [hunk 9]
```diff
@@ -1 +1,7 @@
+# Things you may have to do
+
+- Change the ports in [RobotMap](RobotMap.java)
+- Change the type of MotorController in [Shooter](Shooter.java) (if you change this, you might also need change the
```
Comment (Spikes-2212) [inline src/main/java/frc/robot/subsystems/shooter/README.md]: you might also need **to** change...

**Review by Spikes-2212 — CHANGES_REQUESTED** [review]
change `may` to `might` everywhere

**Review by TurtleMinecraft — APPROVED** [review]

**Pull request merged**